theme-release-110.rst

```diff
@@ -0,0 +1,77 @@
+.. post:: October 14, 2022
+    :tags: theme, release
+    :author: Benjamin
+    :location: Malmö, Sweden
+
+.. meta::
+    :description lang=en:
+        Information on sphinx-rtd-theme version 1.1.0
+
+
+Theme release 1.1.0
+===================
+
+We are happy to announce the release of new version of our theme, `sphinx-rtd-theme`_. In this release, we have focused on bug fixes, backwards compatibility, and making the way for future releases.
+
+.. _sphinx-rtd-theme: https://sphinx-rtd-theme.readthedocs.io/en/stable/
+
+
+Changes and new features 💄
+---------------------------
+
+Visually, we have a couple of small tweaks that most people won't notice unless we mention them here 😇 The general strategy of the theme is to avoid breaking stuff, and that also goes for the visual parts.
```

Comment:
The `breaking stuff` comment is a bit weird without context. I think we can frame it as maintaining backwards compatibility, in a more positive way.  

@benjaoming also 5th reminder to use semantic linebreaks I can comment on lines individually :)

Reply:
Thank you, you are absolutely right, and it was easy to update with a positive phrasing of the same thing.

As for semver.. yes, I try to upgrade my brain. I thought about writing a linter, but it's a bit hard to think up a _good_ syntax for something that's _semantic_ in its nature :)